l10nzhtw.properties

```diff
@@ -0,0 +1,14 @@
+category=Localization
+description=SonarQube Traditional Chinese Language Pack (not compatible with the “Chinese Pack” which uses Simplified Chinese)
+homepageUrl=https://github.com/SonarCommunity/sonar-l10n-zhtw
+archivedVersions=
+publicVersions=1.0
+
+defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
+defaults.mavenArtifactId=sonar-l10n-zhtw-plugin
+
+1.0.description=SonarQube Traditional Chinese Pack
+1.0.sqVersions=[6.0,LATEST]
```

Comment:
Did you mean to start compatibility with 6.0? That's no longer supported by the Marketplace. You _can_ go as low as 6.7 (altho we'll be dropping it from the Marketplace in a few days).
Did you mean 8.0 instead?

Reply:
My SonarQube version is 7.5 and it works with Traditional Chinese plugin. Can I use [7.5, LATEST] instead?

Reply:
I have updated it to [7.5, LATEST], please let me know if there is any problem.